### PR TITLE
Update device.mdx

### PR DESCRIPTION
**Pull request by Nestpebble**
clarified the channel that the LOST_AND_FOUND role uses for messages.

### CODE REVIEW

File: docs/configuration/radio/device.mdx [hunk 1]
```diff
@@ -28,7 +28,7 @@ Acceptable values:
...
-| `LOST_AND_FOUND` |                                                                                                                                                                                                  Lost and Found - Used to automatically send a text message to the mesh with the current position of the device on a frequent interval: "I'm lost! Position: lat / long"                                                                                                                                                                                                   |
+| `LOST_AND_FOUND` |                                                                                                                                                                                                  Lost and Found - Used to automatically send a text message on the primary channel for the device with the current position of the device on a frequent interval: "I'm lost! Position: lat / long"                                                                                                                                                                                                   |
```
Comment (tekstrand) [inline docs/configuration/radio/device.mdx:31]: Just being pedantic but I think `to` was better here.

**Review by tekstrand — APPROVED** [review]
Changes lgtm 🍺

**Review by pdxlocations — DISMISSED** [review]

File: docs/configuration/radio/device.mdx (hunk 1 above)
Comment (pdxlocations) [inline docs/configuration/radio/device.mdx:31]: I was about to merge this but realized you use 'device' twice.  Would you consider:

"Used to automatically send a text message with current position at frequent intervals to the primary channel for the device: "I'm lost! Position: lat / long""

Comment (Nestpebble) [reply docs/configuration/radio/device.mdx:31]: > Just being pedantic but I think `to` was better here.

I'm not fussed either way - I just wanted it clear that it goes on the primary channel!

Comment (Nestpebble) [reply docs/configuration/radio/device.mdx]: Yes, I think I've changed the pull request to this language. Thanks for making it clearer.

**Review by pdxlocations — APPROVED** [review]

**Pull request merged**